pkg/testing/cross_test.go

```diff
@@ -37,6 +38,7 @@ func (suite *CrossTestSuite) TestRecvPacket() {
 		mockSuccessCall = []byte{0x01}
 		mockFailureCall = []byte{0xFF}
 		successMsg      = []byte("mock call succeed")
+		relayerAddr     = common.HexToAddress("wip")
```

Comment:
"wip" is not a hex string

Reply:
fixed at https://github.com/datachainlab/cross-solidity/pull/4/commits/22e6e7f6c2f40130aa4c53635b6b9072aa98fdf9